plugins/modules/df.py

```diff
@@ -0,0 +1,301 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright 2021 Cloudera, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.cloudera.cloud.plugins.module_utils.cdp_common import CdpModule
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: df
+short_description: Enable or Disable CDP DataFlow Services
+description:
+    - Enable or Disable CDP DataFlow Services
+author:
+  - "Dan Chaffelson (@chaffelson)"
+requirements:
+  - cdpy
+options:
+  name:
+    description: The name of the Dataflow Service
+    type: str
+    required: True
+    aliases:
+      - crn
+      - env_crn
+  nodes_min:
+    description: The minimum number of kubernetes nodes needed for the environment. Note that the lowest minimum is 3 nodes.
+    type: int
+    default: 3
+    required: False
+    aliases:
+      - min_k8s_node_count
+  nodes_max:
+    description: The maximum number of  kubernetes  nodes that environment may scale up under high-demand situations.
+    type: int
+    default: 3
+    required: False
+    aliases:
+      - max_k8s_node_count
+  public_loadbalancer:
+    description: Indicates whether or not to use a public load balancer when deploying dependencies stack, such as Nginx Ingress Controller
```

Comment:
Remove the mention of the ingress controller

Reply:
Interestingly, that is taken directly from the CDP CLI documentation. But I have removed it here.